go.mod

```diff
@@ -98,7 +95,7 @@ require (
 	github.com/ipfs/go-ipfs v0.13.1
 	github.com/libp2p/go-libp2p-pubsub v0.7.1
 	github.com/pkg/errors v0.9.1
-	github.com/syndtr/goleveldb v1.0.0
+	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
```

Comment:
Do we need to upgrade the goleveldb? The commit above seems to be from an outdated commit? 

https://github.com/syndtr/goleveldb/commit/2ae1ddf74ef7020251ff1ff0fe8daac21a157761

Reply:
this version gets bumped when i run `go mod tidy`, and if i try to manually put it back to 1.0.0, `go test -v ./...` complains and asks me to run `go mod tidy`. i also tried to `go install github.com/syndtr/goleveldb@v1.0.0` but i get the following:

```
go install github.com/syndtr/goleveldb@v1.0.0
go: github.com/syndtr/goleveldb@v1.0.0: module github.com/syndtr/goleveldb@v1.0.0 found, but does not contain package github.com/syndtr/goleveldb
```

not really sure why this is happening, but figured we could let it be as long as tests are working as expected

Reply:
@alvin-reyes goleveldb seems generally unmaintained fwiw. v1.0.0 was from feb 2019 and the latest commit on master was 4 months ago with failing checks. wonder if we should think about migrating off of it or even coming up with our own ownership story for it https://github.com/syndtr/goleveldb/issues/321